### PR TITLE
Link how to install docker compose v2 from github releases

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
This links docker compose GitHub repositories instruction  on how to install from GH releases (preferable for Compose V2 on Linux / Mac systems or whenever not using Docker for Desktop). It basically boils down to
```bash
DOCKER_COMPOSE_VERSION=$(curl --silent https://api.github.com/repos/docker/compose/releases/latest | jq .name -r)
DESTINATION=${HOME}/.docker/cli-plugins
mkdir -p ${DESTINATION}
curl -L "https://github.com/docker/compose/releases/download/$DOCKER_COMPOSE_VERSION/docker-compose-$(uname -s)-$(uname -m)" -o ${DESTINATION}/docker-compose
chmod +x ${DESTINATION}/docker-compose
```
but linking upstream resource seems preferable.

### CODE REVIEW

**Review by cliveseldon — APPROVED** [review]

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[cliveseldon](https://github.com/SeldonIO/seldon-core/pull/4516#pullrequestreview-1223248494 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/v2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[cliveseldon](https://github.com/SeldonIO/seldon-core/pull/4516#pullrequestreview-1223248494 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/v2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/source/contents/getting-started/docker-installation/index.md
Comment (agrski) [inline docs/source/contents/getting-started/docker-installation/index.md:7]:
```suggestion
 3. Install [Docker Compose](https://docs.docker.com/compose/install/) (or directly from GitHub [release](https://github.com/docker/compose#linux) if not using Docker Desktop).
```
:question: This is the product name, right?

**Pull request merged**
